packages/playwright/src/reporters/html.ts

```diff
@@ -372,15 +373,15 @@ class HtmlBuilder {
   }
 
   private async _writeReportData(filePath: string) {
-    fs.appendFileSync(filePath, '<script>\nwindow.playwrightReportBase64 = "data:application/zip;base64,');
+    fs.appendFileSync(filePath, '<script id="playwrightReportBase64" type="application/zip">data:application/zip;base64,');
     await new Promise(f => {
       this._dataZipFile!.end(undefined, () => {
         this._dataZipFile!.outputStream
             .pipe(new Base64Encoder())
             .pipe(fs.createWriteStream(filePath, { flags: 'a' })).on('close', f);
       });
     });
-    fs.appendFileSync(filePath, '";</script>');
+    fs.appendFileSync(filePath, '"</script>');
```

Comment:
```suggestion
    fs.appendFileSync(filePath, '</script>');
```
what is this quote for?